comp/netflow/server/server.go

```diff
@@ -0,0 +1,111 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+package server
+
+import (
+	"errors"
+	"net/http"
+	"time"
+
+	"github.com/DataDog/datadog-agent/comp/core/log"
+	"github.com/DataDog/datadog-agent/comp/netflow/config"
+	"github.com/DataDog/datadog-agent/comp/netflow/flowaggregator"
+	"github.com/DataDog/datadog-agent/comp/netflow/forwarder"
+	"github.com/DataDog/datadog-agent/comp/netflow/hostname"
+	"github.com/DataDog/datadog-agent/comp/netflow/sender"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"go.uber.org/fx"
+)
+
+type dependencies struct {
+	fx.In
+	Config    config.Component
```

Comment:
`Config  config.Component` often refers to `comp/config` component. Can you either change the name of the field or the type to remove the ambiguity with `comp/config`?

Reply:
Done.